### PR TITLE
DevDocs: Add Animate component

### PR DESCRIPTION
**Pull request by andfinally**
This PR partially addresses #24136 - a lot of components and blocks aren't listed in DevDocs.

* Added `Animate` component to DevDocs and DevDocs playground.
* Reordered import statements in `playground.jsx`.

cc @scruffian

### CODE REVIEW

File: client/devdocs/design/playground.jsx [hunk 1]
```diff
@@ -88,6 +89,9 @@ import JetpackColophon from 'components/jetpack-colophon';
 import JetpackLogo from 'components/jetpack-logo';
 import LanguagePicker from 'components/language-picker';
 import ListEnd from 'components/list-end';
+import NavItem from 'components/section-nav/item';
+import NavSegmented from 'components/section-nav/segmented';
+import NavTabs from 'components/section-nav/tabs';
```
Comment (scruffian) [inline client/devdocs/design/playground.jsx]: Seems like lots of these should belong in a different PR

Comment (andfinally) [reply client/devdocs/design/playground.jsx]: Oh right, OK Ben, I can amend - this was reordering the existing imports alphabetically - I've got the same change in my other PRs.

**Comment by rodrigoi** [comment]
since `type` has a limited number of values (_appear_ and _fade-in_ apparently) it may be a good idea to add them to the `README.md` file.
Otherwise, the example looks great and the component works as expected on the playground.

**Comment by scruffian** [comment]
I fixed this up. It could use another review...

**Comment by andfinally** [comment]
Cheers Ben - I'm still earning my stripes so not approving PRs, but LGTM!
